jasync-sql/src/main/java/io/micronaut/configuration/jasync/condition/package-info.java

```diff
@@ -21,4 +21,4 @@
  * @since 1.0
  */
 
-package io.micronaut.configuration.postgres.reactive.condition;
+package io.micronaut.configuration.jasync.condition;
```

Comment:
Seems that some packages were renamed? These probably shouldn't be part of this change

Reply:
I just renamed packages that are copied for postgresql reactive. I think it should be fine, isn't it?